tests/test_PolarBasis2D.py

```diff
@@ -0,0 +1,127 @@
+import numpy as np
+from unittest import TestCase
+
+from aspire.basis.polar_2d import PolarBasis2D
+
+
+class PolarBasis2DTestCase(TestCase):
```

Comment:
Can we add a test to verify that `evaluate_t` is the adjoint of `evaluate`?

Reply:
Do you mean that `evaluate_t` has the similar result of  `expand`?  Or x = evaluate_t(evaluate(x))?  I checked it yesterday, x != evaluate_t(evaluate(x)).

Reply:
In higher reslution (larger nrad and ntheta), x looks closer to evaluate_t(evaluate(x)). Probably we need to use a disk mask, but this will not reproduce the results from previous code. I think it is OK for common line and orientation estimation but not accurate enough for image expanding.

Reply:
in the new commit fefb446, a new function was added to test whether the `evaluate` is the adjoint operator of `evaluate_t` , namely satisfying  (y, A*x) = (A^t*y,x) = (B*y, x).